.github/workflows/pkgci_test_sharktank.yml

```
@@ -25,9 +25,15 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          # CPU
-          - name: cpu_llvm_task
-            runs-on: ubuntu-20.04
+          - name: cpu_task
+            target: target_cpu
+            gpu: none
+            runs-on: nodai-amdgpu-w7900-x86-64
```

Can the cpu tests continue to run on github-hosted runners? Unless the tests are too slow (20+ minutes) or require large file downloads (2 minutes+ downloading, or bandwidth that we pay for and would rather have cached), prefer to use github-hosted runners to help with job reliability and allow this job to run in forks.

```suggestion
          - name: cpu_task
            target: target_cpu
            gpu: none
            runs-on: ubuntu-20.04
```

---

More context in our docs: https://iree.dev/developers/general/github-actions/#using-github-hosted-and-self-hosted-runners